addons/account/models/ir_module.py

```diff
@@ -78,6 +78,7 @@ def try_loading(env):
                 env['account.chart.template'].try_loading(
                     guessed,
                     env.company,
+                    force_create=True
```

Comment:
This is the default, any reason why you add it explicitly?

Reply:
Modified :)